main.py

```diff
@@ -1,10 +1,52 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as players_file:
+        players = json.load(players_file)
+
+    for player in players:
+        nickname = player
```

Comment:
There is no need to assign the value of one variable directly to another variable. Perform an interaction using `.items()` method
```suggestion
    for nickname, player_data in players.items():
```